charts/camunda-platform-alpha/values.yaml

```diff
@@ -1324,7 +1324,7 @@ operate:
     automountServiceAccountToken: false
 
   ingress:
-    ## @param operate.ingress.enabled if true, an ingress resource is deployed with the Operate deployment. Only useful if an ingress controller is available, like nginx.
+    ## @param operate.ingress.enabled if true, an ingress resource is deployed with the Operate deployment. Only useful if an ingress controller is available, like nginx. Warning, separated ingress is deprecated and will be removed in the next release, please use global.ingress instead.
```

Comment:
Warning: This parameter to configure Separated ingress is deprecated and will be removed in the next release. Please use Combined ingress by setting global.ingress: true.

Reply:
let apply this text to all other component parameters